e2e/test/scenarios/sharing/subscriptions.cy.spec.js

```diff
@@ -107,6 +107,22 @@ describe("scenarios > dashboard > subscriptions", () => {
         cy.findByText("Emailed hourly");
       });
 
+      it("should not add a recipient when Escape is pressed (metabase#24629)", () => {
+        openDashboardSubscriptions(ORDERS_DASHBOARD_ID);
+        // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+        cy.findByText("Email it").click();
```

Comment:
Wdyt about `H.sidebar().findByText("Email it").click();` to remove eslint-disable?